src/main/java/org/opentripplanner/routing/graph/Graph.java

```diff
@@ -197,6 +198,15 @@
 
     /** True if GTFS data was loaded into this Graph. */
     public boolean hasTransit = false;
+    private ClusterList clusters;
```

Comment:
Właściwie to nie potrzebujemy zapisywać ponownie listy klastrów w grafie - ona jest potrzebna tylko tymczasowo w `TrafifcPredictionBuilderModule`